internal/rpcserver/rpcserverhelp.go

```diff
@@ -510,6 +510,15 @@ var helpDescsEnUS = map[string]string{
 	// GetMiningInfoCmd help.
 	"getmininginfo--synopsis": "Returns a JSON object containing mining-related information.",
 
+	// GetMixMessage help.
+	"getmixmessage--synopsis": "Returns a mix message if it is currently accepted by the mixpool.",
```

Comment:
Just a minor suggestion, but:

```suggestion
	"getmixmessage--synopsis": "Returns a mix message given its hash if it is currently in the mixpool.",
```

The reason I suggest that change in wording is because it reads as if you can query it at any point after its been accepted to me as is.  I think the suggested change makes it pretty obvious it only applies while it is still in the pool.

Reply:
well, it does say "currently accepted", but i could see the confusion.

I don't like simply saying that it is "in" the mixpool, because mixpool also saves the orphan KEs (and other orphan session messages, if the KE is unknown). they are "in" the mixpool, but the query will fail.

Reply:
Well, I guess for me orphan mix messages are not actually in the mixpool, in the same way orphan transactions aren't in the mempool, rather they are staged to potentially be added to it if there parents show up, but I can see your point.

It's not really a big deal, so I'm fine with leaving it as it is.